truck_inspection_client/src/truck_inspection_client.cpp

```diff
@@ -36,6 +36,7 @@ namespace truck_inspection_client
         pnh_->getParam("permit_required", permit_required_);
         pnh_->getParam("pre_trip_ads_health_check", pre_trip_ads_health_check_);
         mo_pub_ = nh_->advertise<cav_msgs::MobilityOperation>("outgoing_mobility_operation", 5);
+        vin_system_alert_pub_ = nh_->advertise<cav_msgs::SystemAlert>("system_alert", 10);
```

Comment:
It is not necessary to create your own system_alert publisher. CARMANodeHandle does that by default.
You can just call
nh_->publishSystemAlert(alert_msg);

Reply:
Addressed: https://github.com/usdot-fhwa-stol/carma-platform/pull/1020/commits/56dd79335ca2255d024912c27d7e13719a88414d